app/jobs/characterize_job.rb

```diff
@@ -2,9 +2,10 @@ class CharacterizeJob < ActiveJob::Base
   queue_as CurationConcerns.config.ingest_queue_name
 
   # @param [FileSet] file_set
-  # @param [String] filename a local path for the file to characterize. By using this, we don't have to pull a copy out of fedora.
+  # @param [String] filename a local path for the file to characterize so we don't have to pull a copy out of fedora.
   def perform(file_set, filename)
-    Hydra::Works::CharacterizationService.run(file_set, filename)
+    return unless file_set.characterization_proxy.content
```

Comment:
Yes, I see. Alternatively, we can check to see if the proxy is a `NullCharacterizationProxy`. This is what gets set if the relation (i.e. original_file) isn't there. Would it make sense to have the `characterization_proxy?` method in here:
https://github.com/projecthydra-labs/curation_concerns/blob/master/app/models/concerns/curation_concerns/characterization.rb#L17-30